packages/elements/src/components/button/__styles__/index.tsx

```diff
@@ -113,13 +141,52 @@ export const ElButton = styled.button`
       }
     }
   }
+
+  ${isMobile} {
+    height: 2.5rem;
+    font-size: 1rem;
+    padding: ${buttonYPaddingMobile}rem 0.75rem;
+  }
+
+  &.el-floating-button {
+    border-radius: 100%;
+    height: 3.75rem;
+    width: 3.75rem;
+    margin: 0.5rem;
+
+    padding: 0.4rem;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+
+    ${isMobile} {
+      width: 2.5rem;
+      height: 2.5rem;
+      margin: 0.5rem;
+    }
+
+    &${elIsLoading} {
```

Comment:
Needs a `.` ie `&.${elIsLoading}`

Reply:
I don't quite get this one. Sometimes I add a dot sometimes I don't 🤷‍♀️   in this case, if I add the dot, the applied styles disappear 🤷‍♀️